File: IKeep.Lib/Models/FormatLabel.cs
using IKeep.Lib.Core;
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Text;

namespace IKeep.Lib.Models
{
    public class FormatLabel : Entity
    {
        public string Name { get; set; }
        public string Extension { get; set; }
        [JsonIgnore]
        public virtual ICollection<FormatValue> FormatValues { get; set; }
        [JsonIgnore]
        public virtual ICollection<GenericChoreFormatLabel> GenericChoreFormatLabels { get; set; }
    }
}

File: IKeep.Web/ClientApp/src/models/FormatLabel.js
import Entity from './Entity'

class FormatLabel extends Entity
{
    constructor(json)
    {
        super(json);
        if(json)
        {
            this.Name = json.name;
            this.Extension = json.extension;
        }
        else
        {
            this.Name = "";
            this.Extension = "";
        }

    }
}

export default FormatLabel;